src/routes/auth-token/create.js

```diff
@@ -36,17 +28,16 @@ export default {
 
   handler(request, response) {
 
-    // see: https://hackernoon.com/never-use-passwords-again-with-ethereum-and-metamask-b61c7e409f0d
+    // see: https://medium.com/metamask/scaling-web3-with-signtypeddata-91d6efc8b290
 
     try {
 
-      const { v, r, s } = ethUtil.fromRpcSig(ethUtil.toBuffer(request.parameters.signedData))
-      const publicKey = ethUtil.ecrecover(hashedPersonalMessage, v, r, s)
-
-      const userAddressBuffer = ethUtil.toBuffer(request.parameters.userAddress)
-      const senderAddressBuffer = ethUtil.publicToAddress(publicKey)
+      const recoveredAddress = ethSigUtil.recoverTypedSignature({
+        data: typedDataToSign,
```

Comment:
Can you flip these two lines so they're in descending order by length, i.e.:

```javascript
const recoveredAddress = ethSigUtil.recoverTypedSignature({
  sig: request.parameters.signedData,
  data: config.typedDataToSign,
})
```

Reply:
Happy to. Is `longest parameter name first` part of our style guide? Haven't come across that before :)